modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -87,18 +93,20 @@ const ClassicEditor = ({
 		};
 	}, [contents, getHTML, name]);
 
-	useEventListener(
-		'resize',
-		() => setToolbarSet(getToolbarSet(initialToolbarSet)),
-		true,
-		window
-	);
+	const onResize = debounce(() => {
+		setToolbarSet(getToolbarSet(initialToolbarSet));
+	}, 200);
```

Comment:
No need to send another rev to this, but for future reference, I usually name my magical constants to give them the illusion of non-arbitrariness.

```js
const SCIENTIFICALLY_PROVEN_OPTIMAL_DEBOUNCE_DELAY_IN_MS = 200;
```

😉